packages/geo/src/lib/draw/shared/draw-style.service.ts

```diff
@@ -10,160 +11,176 @@ import { MapService } from '../../map/shared/map.service';
   })
 export class DrawStyleService {
 
-    private fillColor: string = 'rgba(255,255,255,0.4)';
-    private strokeColor: string = 'rgba(51,153,204,1)';
-    private drawCounter: number = 1;
-    private toggleLabel = true;
-    private icon;
-
-    constructor(
-      private mapService: MapService
-    ) {}
-
-    getFill(): string {
-        return this.fillColor;
-    }
-
-    setFill(fillColor: string) {
-        this.fillColor = fillColor;
-    }
-
-    getStroke(): string {
-        return this.strokeColor;
+  private fillColor: OlColor = 'rgba(255,255,255,0.4)';
+  private strokeColor: OlColor = 'rgba(143,7,7,1)';
+  private strokeWidth: number = 1;
+  private drawCount: number = 1;
+  private labelsAreShown = true;
+  private icon: string;
+
+  constructor(
+    private mapService: MapService
+  ) {}
+
+  getFillColor(): OlColor {
+    return this.fillColor;
+  }
+
+  setFillColor(fillColor: OlColor) {
+    this.fillColor = fillColor;
+  }
+
+  getStrokeColor(): OlColor {
+    return this.strokeColor;
+  }
+
+  setStrokeColor(strokeColor: OlColor) {
```

Comment:
validate if all the following method are used...

Reply:
> 1- Split your differents modification into commit. issueX = 1 or more commit. Your PR is too broad and it is hard to review.
Gonna take that into account in the future!